### DISCUSSION TITLE
Installing core_9.0.0_amd64.deb on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by awolff64** [coreemu/core#717] [Q&A] [answered]
I try to install core on a freshly installed Ubuntu 22.04 machine according to the instructions from the deb-package.
apt install -y ./core_9.0.0_amd64.dev

I also installed pzthon3-venv, python3-pip, python3-tk, phython3-bs4

I fail - installation runs ok but ...
... when running core-gui I get an error
    ...
    from google.protobuf.internal import builder as _builder

I am afraid, I missed something fundamental.
BTW: can't run it on 20.04 as this does not provide the required python > 3.9 out of the box.

Can somebody help?

**Comment by awolff64** [comment]
looks like there is no more a builder in googles protobuf  (4.21.9) in python3.10

**Reply by awolff64** [reply]
the import is in grpc/common_pb2.py which seems a file generated by some tool

**Comment by bharnden** [comment]
I missed properly updating a dependency for the install, looking to correct now and provide an updated release, thanks.

**Reply by awolff64** [reply]
Some how the installed protobuf and and grpcio.protoc do not match.
Interestingly 
https://github.com/protocolbuffers/protobuf/tree/main/python/google/protobuf/internal
has the file builder.py. However the installed protobuf does not have/provide it.

Currently blocks me to set up my emulation.
Help appreciated.

Thanks,
Andreas

**Comment by awolff64** [comment, marked as answer]
Fixed in 9.0,1 - many thanks!
